run_all_trials.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+TEAM_NAME=$1
+
+#TODO: get list of trials automatically
+declare -a TRIAL_NAMES=("example_trial1" "example_trial2")
```

Comment:
thank you for your advice here (and elsewhere!) -- indeed that looks like a simpler way to achieve it